charts/envoy/values.yaml

```diff
@@ -114,3 +114,21 @@ tolerations: []
 
 # affinity -- the affinity/anti-affinity feature, greatly expands the types of constraints you can express
 affinity: {}
+
+tls:
+  # -- TLS configuration between client and Envoy.
+  downstreamTls:
```

Comment:
nit: Since it is clear from the upper field name that this is a TLS setting, simply `downstream` may be acceptable.

Reply:
Ah, I see. I agree. It's a duplicated field name.
I fixed it in 32f12fd26b75d3dd13d67f6004f1defa24662ee4.